src/renderer/views/Trending/Trending.css

```diff
@@ -14,10 +14,17 @@
   width: 100%;
   display: grid;
   grid-template-columns: 1fr 1fr 1fr 1fr;
+  margin-top: -3px;
+  color: var(--tertiary-text-color);
```

Comment:
Is there a reason why the color is being changed here and not .tab like in Channel.css?

Reply:
You know, I didn't really think much about it. Feel free to change this in my PR if you so please.

Reply:
It's not a big deal, just a massive nitpick on my end